From 22cb634148b70ee2474aa6d63d609a26c0f8461f Mon Sep 17 00:00:00 2001
From: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
Date: Tue, 30 Jun 2020 12:24:57 +0100
Subject: [PATCH 33/51] DB820c: Correctly move DB820c to Qualcomm/apq8096

DB820c belongs to APQ8096 SoC, so it should be moved to ucm2/apq8096
folder rather than apq8016.

So correctly move it to apq8096 to reflect the correct SoC.

Fixes: 73c105b4707b ("DB820c/DB845c: move to Qualcomm/ tree")
Signed-off-by: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/Qualcomm/apq8016/HDMI.conf    |  33 ------------
 ucm2/Qualcomm/apq8016/HiFi.conf    | 100 -------------------------------------
 ucm2/Qualcomm/apq8016/apq8016.conf |  11 ----
 ucm2/Qualcomm/apq8096/HDMI.conf    |  33 ++++++++++++
 ucm2/Qualcomm/apq8096/HiFi.conf    | 100 +++++++++++++++++++++++++++++++++++++
 ucm2/Qualcomm/apq8096/apq8096.conf |  11 ++++
 ucm2/module/snd_soc_apq8096.conf   |   2 +-
 7 files changed, 145 insertions(+), 145 deletions(-)
 delete mode 100644 ucm2/Qualcomm/apq8016/HDMI.conf
 delete mode 100644 ucm2/Qualcomm/apq8016/HiFi.conf
 delete mode 100644 ucm2/Qualcomm/apq8016/apq8016.conf
 create mode 100644 ucm2/Qualcomm/apq8096/HDMI.conf
 create mode 100644 ucm2/Qualcomm/apq8096/HiFi.conf
 create mode 100644 ucm2/Qualcomm/apq8096/apq8096.conf

diff --git a/ucm2/Qualcomm/apq8016/HDMI.conf b/ucm2/Qualcomm/apq8016/HDMI.conf
deleted file mode 100644
index 5bdf07eac5c9..000000000000
--- a/ucm2/Qualcomm/apq8016/HDMI.conf
+++ /dev/null
@@ -1,33 +0,0 @@
-# Use case configuration for DB820c board.
-# Author: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
-
-SectionVerb {
-	EnableSequence [
-		cset "name='HDMI Mixer MultiMedia1' 1"
-	]
-
-	DisableSequence [
-		cset "name='HDMI Mixer MultiMedia1' 0"
-	]
-
-	Value {
-		TQ "HiFi"
-	}
-}
-
-SectionDevice."HDMI" {
-	Comment "HDMI Digital Stereo Output"
-
-	EnableSequence [
-		cset "name='HDMI Mixer MultiMedia1' 1"
-	]
-
-	DisableSequence [
-		cset "name='HDMI Mixer MultiMedia1' 0"
-	]
-
-	Value {
-		PlaybackPriority 200
-		PlaybackPCM "plughw:${CardId}"
-	}
-}
diff --git a/ucm2/Qualcomm/apq8016/HiFi.conf b/ucm2/Qualcomm/apq8016/HiFi.conf
deleted file mode 100644
index 89280d72450c..000000000000
--- a/ucm2/Qualcomm/apq8016/HiFi.conf
+++ /dev/null
@@ -1,100 +0,0 @@
-# Use case configuration for DB820c board.
-# Author: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
-
-SectionVerb {
-
-	EnableSequence [
-		cset "name='SLIM RX0 MUX' ZERO"
-		cset "name='SLIM RX1 MUX' ZERO"
-		cset "name='SLIM RX2 MUX' ZERO"
-		cset "name='SLIM RX3 MUX' ZERO"
-		cset "name='SLIM RX4 MUX' ZERO"
-		cset "name='SLIM RX5 MUX' AIF4_PB"
-		cset "name='SLIM RX6 MUX' AIF4_PB"
-		cset "name='SLIM RX7 MUX' ZERO"
-		cset "name='RX INT1_2 MUX' RX5"
-		cset "name='RX INT2_2 MUX' RX6"
-		## gain to  0dB
-		cset "name='RX5 Digital Volume' 68"
-		## gain to  0dB
-		cset "name='RX6 Digital Volume' 68"
-		cset "name='SLIMBUS_6_RX Audio Mixer MultiMedia2' 1"
-		cset "name='MultiMedia3 Mixer SLIMBUS_0_TX' 1"
-		cset "name='RX INT1 DEM MUX' CLSH_DSM_OUT"
-		cset "name='RX INT2 DEM MUX' CLSH_DSM_OUT"
-		cset "name='AIF1_CAP Mixer SLIM TX0' 1"
-		cset "name='SLIM TX0 MUX' DEC0"
-		cset "name='ADC2 Volume' 12"
-		cset "name='ADC MUX0' AMIC"
-		cset "name='AMIC MUX0' ADC2"
-	]
-
-	DisableSequence [
-		cset "name='SLIMBUS_6_RX Audio Mixer MultiMedia2' 0"
-		cset "name='MultiMedia3 Mixer SLIMBUS_0_TX' 0"
-	]
-}
-
-SectionDevice."Headphones" {
-	Comment "Headphones playback"
-
-	EnableSequence [
-		cset "name='SLIM RX0 MUX' ZERO"
-		cset "name='SLIM RX1 MUX' ZERO"
-		cset "name='SLIM RX2 MUX' ZERO"
-		cset "name='SLIM RX3 MUX' ZERO"
-		cset "name='SLIM RX4 MUX' ZERO"
-		cset "name='SLIM RX5 MUX' AIF4_PB"
-		cset "name='SLIM RX6 MUX' AIF4_PB"
-		cset "name='SLIM RX7 MUX' ZERO"
-		cset "name='RX INT1_2 MUX' RX5"
-		cset "name='RX INT2_2 MUX' RX6"
-		## gain to  0dB
-		cset "name='RX5 Digital Volume' 68"
-		## gain to  0dB
-		cset "name='RX6 Digital Volume' 68"
-		cset "name='SLIMBUS_6_RX Audio Mixer MultiMedia2' 1"
-		cset "name='RX INT1 DEM MUX' CLSH_DSM_OUT"
-		cset "name='RX INT2 DEM MUX' CLSH_DSM_OUT"
-	]
-
-	DisableSequence [
-		cset "name='RX5 Digital Volume' 0"
-		cset "name='RX6 Digital Volume' 0"
-		cset "name='SLIM RX5 MUX' ZERO"
-		cset "name='SLIM RX6 MUX' ZERO"
-		cset "name='SLIMBUS_6_RX Audio Mixer MultiMedia2' 0"
-	]
-
-	Value {
-		PlaybackPriority 100
-		PlaybackPCM "plughw:${CardId},1"
-	}
-}
-
-SectionDevice."Headset" {
-	Comment "Headset Microphone"
-
-	EnableSequence [
-		cset "name='MultiMedia3 Mixer SLIMBUS_0_TX' 1"
-		cset "name='AIF1_CAP Mixer SLIM TX0' 1"
-		cset "name='SLIM TX0 MUX' DEC0"
-		cset "name='ADC2 Volume' 12"
-		cset "name='ADC MUX0' AMIC"
-		cset "name='AMIC MUX0' ADC2"
-	]
-
-	DisableSequence [
-		cset "name='MultiMedia3 Mixer SLIMBUS_0_TX' 0"
-		cset "name='AIF1_CAP Mixer SLIM TX0' 0"
-		cset "name='AMIC MUX0' ZERO"
-		cset "name='SLIM TX0 MUX' ZERO"
-		cset "name='ADC2 Volume' 0"
-	]
-
-	Value {
-		CapturePriority 100
-		CapturePCM "plughw:${CardId},2"
-		CaptureChannels 1
-	}
-}
diff --git a/ucm2/Qualcomm/apq8016/apq8016.conf b/ucm2/Qualcomm/apq8016/apq8016.conf
deleted file mode 100644
index 538c7984e180..000000000000
--- a/ucm2/Qualcomm/apq8016/apq8016.conf
+++ /dev/null
@@ -1,11 +0,0 @@
-Syntax 2
-
-SectionUseCase."HiFi" {
-	File "/Qualcomm/apq8016/HiFi.conf"
-	Comment "HiFi quality Music"
-}
-
-SectionUseCase."HDMI" {
-	File "/Qualcomm/apq8016/HDMI.conf"
-	Comment "HDMI output"
-}
diff --git a/ucm2/Qualcomm/apq8096/HDMI.conf b/ucm2/Qualcomm/apq8096/HDMI.conf
new file mode 100644
index 000000000000..5bdf07eac5c9
--- /dev/null
+++ b/ucm2/Qualcomm/apq8096/HDMI.conf
@@ -0,0 +1,33 @@
+# Use case configuration for DB820c board.
+# Author: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
+
+SectionVerb {
+	EnableSequence [
+		cset "name='HDMI Mixer MultiMedia1' 1"
+	]
+
+	DisableSequence [
+		cset "name='HDMI Mixer MultiMedia1' 0"
+	]
+
+	Value {
+		TQ "HiFi"
+	}
+}
+
+SectionDevice."HDMI" {
+	Comment "HDMI Digital Stereo Output"
+
+	EnableSequence [
+		cset "name='HDMI Mixer MultiMedia1' 1"
+	]
+
+	DisableSequence [
+		cset "name='HDMI Mixer MultiMedia1' 0"
+	]
+
+	Value {
+		PlaybackPriority 200
+		PlaybackPCM "plughw:${CardId}"
+	}
+}
diff --git a/ucm2/Qualcomm/apq8096/HiFi.conf b/ucm2/Qualcomm/apq8096/HiFi.conf
new file mode 100644
index 000000000000..89280d72450c
--- /dev/null
+++ b/ucm2/Qualcomm/apq8096/HiFi.conf
@@ -0,0 +1,100 @@
+# Use case configuration for DB820c board.
+# Author: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
+
+SectionVerb {
+
+	EnableSequence [
+		cset "name='SLIM RX0 MUX' ZERO"
+		cset "name='SLIM RX1 MUX' ZERO"
+		cset "name='SLIM RX2 MUX' ZERO"
+		cset "name='SLIM RX3 MUX' ZERO"
+		cset "name='SLIM RX4 MUX' ZERO"
+		cset "name='SLIM RX5 MUX' AIF4_PB"
+		cset "name='SLIM RX6 MUX' AIF4_PB"
+		cset "name='SLIM RX7 MUX' ZERO"
+		cset "name='RX INT1_2 MUX' RX5"
+		cset "name='RX INT2_2 MUX' RX6"
+		## gain to  0dB
+		cset "name='RX5 Digital Volume' 68"
+		## gain to  0dB
+		cset "name='RX6 Digital Volume' 68"
+		cset "name='SLIMBUS_6_RX Audio Mixer MultiMedia2' 1"
+		cset "name='MultiMedia3 Mixer SLIMBUS_0_TX' 1"
+		cset "name='RX INT1 DEM MUX' CLSH_DSM_OUT"
+		cset "name='RX INT2 DEM MUX' CLSH_DSM_OUT"
+		cset "name='AIF1_CAP Mixer SLIM TX0' 1"
+		cset "name='SLIM TX0 MUX' DEC0"
+		cset "name='ADC2 Volume' 12"
+		cset "name='ADC MUX0' AMIC"
+		cset "name='AMIC MUX0' ADC2"
+	]
+
+	DisableSequence [
+		cset "name='SLIMBUS_6_RX Audio Mixer MultiMedia2' 0"
+		cset "name='MultiMedia3 Mixer SLIMBUS_0_TX' 0"
+	]
+}
+
+SectionDevice."Headphones" {
+	Comment "Headphones playback"
+
+	EnableSequence [
+		cset "name='SLIM RX0 MUX' ZERO"
+		cset "name='SLIM RX1 MUX' ZERO"
+		cset "name='SLIM RX2 MUX' ZERO"
+		cset "name='SLIM RX3 MUX' ZERO"
+		cset "name='SLIM RX4 MUX' ZERO"
+		cset "name='SLIM RX5 MUX' AIF4_PB"
+		cset "name='SLIM RX6 MUX' AIF4_PB"
+		cset "name='SLIM RX7 MUX' ZERO"
+		cset "name='RX INT1_2 MUX' RX5"
+		cset "name='RX INT2_2 MUX' RX6"
+		## gain to  0dB
+		cset "name='RX5 Digital Volume' 68"
+		## gain to  0dB
+		cset "name='RX6 Digital Volume' 68"
+		cset "name='SLIMBUS_6_RX Audio Mixer MultiMedia2' 1"
+		cset "name='RX INT1 DEM MUX' CLSH_DSM_OUT"
+		cset "name='RX INT2 DEM MUX' CLSH_DSM_OUT"
+	]
+
+	DisableSequence [
+		cset "name='RX5 Digital Volume' 0"
+		cset "name='RX6 Digital Volume' 0"
+		cset "name='SLIM RX5 MUX' ZERO"
+		cset "name='SLIM RX6 MUX' ZERO"
+		cset "name='SLIMBUS_6_RX Audio Mixer MultiMedia2' 0"
+	]
+
+	Value {
+		PlaybackPriority 100
+		PlaybackPCM "plughw:${CardId},1"
+	}
+}
+
+SectionDevice."Headset" {
+	Comment "Headset Microphone"
+
+	EnableSequence [
+		cset "name='MultiMedia3 Mixer SLIMBUS_0_TX' 1"
+		cset "name='AIF1_CAP Mixer SLIM TX0' 1"
+		cset "name='SLIM TX0 MUX' DEC0"
+		cset "name='ADC2 Volume' 12"
+		cset "name='ADC MUX0' AMIC"
+		cset "name='AMIC MUX0' ADC2"
+	]
+
+	DisableSequence [
+		cset "name='MultiMedia3 Mixer SLIMBUS_0_TX' 0"
+		cset "name='AIF1_CAP Mixer SLIM TX0' 0"
+		cset "name='AMIC MUX0' ZERO"
+		cset "name='SLIM TX0 MUX' ZERO"
+		cset "name='ADC2 Volume' 0"
+	]
+
+	Value {
+		CapturePriority 100
+		CapturePCM "plughw:${CardId},2"
+		CaptureChannels 1
+	}
+}
diff --git a/ucm2/Qualcomm/apq8096/apq8096.conf b/ucm2/Qualcomm/apq8096/apq8096.conf
new file mode 100644
index 000000000000..3a26c5ed5f29
--- /dev/null
+++ b/ucm2/Qualcomm/apq8096/apq8096.conf
@@ -0,0 +1,11 @@
+Syntax 2
+
+SectionUseCase."HiFi" {
+	File "/Qualcomm/apq8096/HiFi.conf"
+	Comment "HiFi quality Music"
+}
+
+SectionUseCase."HDMI" {
+	File "/Qualcomm/apq8096/HDMI.conf"
+	Comment "HDMI output"
+}
diff --git a/ucm2/module/snd_soc_apq8096.conf b/ucm2/module/snd_soc_apq8096.conf
index 9afc13c79031..2470d93fb597 120000
--- a/ucm2/module/snd_soc_apq8096.conf
+++ b/ucm2/module/snd_soc_apq8096.conf
@@ -1 +1 @@
-../Qualcomm/apq8016/apq8016.conf
\ No newline at end of file
+../Qualcomm/apq8096/apq8096.conf
\ No newline at end of file
-- 
2.16.4

